src/static/Services.js

```diff
@@ -0,0 +1,26 @@
+
+export const API = {
+    fetchAllLabs:  async ()  => {
+        const endpointAPI = "https://match-labs-api.herokuapp.com/api/no_auth/labs";
+        const response = await fetch(endpointAPI);
+        const responseJson = await response.json();
+
+        return responseJson;
+    },
+
+    fetchLab:  async (labId)  => {
+        const endpointAPI = `https://match-labs-api.herokuapp.com/api/no_auth/labs/${labId}`;
```

Comment:
One optimization, you could store the API url in a 
`const baseUrl  = https://match-labs-api.herokuapp.com/api/no_auth/`
and then just use that in order to avoid repetition. But functionality wise, it would work the same.